layouts/community/ortho_4x12/jotix/keymap.c

```diff
@@ -10,83 +7,178 @@ extern keymap_config_t keymap_config;
  * |        |   F1   |   F2   |   F3   |   F4   |   F5   |   F6   |    -   |    =   |    [   |    ]   |    \   |
  * +--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
- * |        |   F7   |   F8   |   F9   |   F10  |   F11  |   F12  |    _   |    +   |    {   |    }   |    |   |
+ * |        |   F7   |   F8   |   F9   |   F10  |   F11  |   F12  |    `   |        |        |        |        |
  * +--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
  * |        |        |        |        |        |        |        |        |  home  |  pgdn  |  pgun  |   end  |
  * +--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+
  */
-[_RAISE] = LAYOUT_ortho_4x12 (
-    KC_TILD, KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_DEL,
+[_FN] = LAYOUT_ortho_4x12 (
+    _______, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    _______,
     _______, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS,
-    _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_UNDS, KC_PLUS, KC_LCBR, KC_RCBR, KC_PIPE,
+    _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_GRV,  _______, _______, _______, _______,
     _______, _______, _______, _______, _______, _______, _______, _______, KC_HOME, KC_PGDN, KC_PGUP, KC_END
 ),
 
-/* Adjust */
+/* 
+ * unicode
+ */
+[_UNICODE] = LAYOUT_ortho_4x12 ( 
+    _______, X(EXCL), X(WORR), E_UNIC,  X(EURO), X(TONG), Y_UNIC,  U_UNIC,  I_UNIC,  O_UNIC,  X(POUN), _______,
+    _______, A_UNIC,  X(SMIL), X(DISS), X(SCRE), X(DEGR), X(SMIH), X(NOT),  X(QUAR), X(HALF), X(ORDF), X(ORDM),
+    _______, X(DIZY), X(ANGR), X(COPY), X(QUAD), X(CUBE), N_UNIC,  X(NEUT), X(LDQU), X(RDQU), X(QUES), _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, X(ARRL), X(ARRD), X(ARRU), X(ARRR)
+),
+
+/* 
+ * Adjust 
+ */
 [_ADJUST] = LAYOUT_ortho_4x12 (
     _______, RESET,   _______, _______, _______, _______, _______, _______, KC_PSCR, KC_SLCK, KC_PAUS, _______,
     _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
-    _______, _______, _______, KC_CAPS, _______, BL_STEP, BL_BRTG, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, BL_STEP, BL_BRTG, _______, _______, _______, _______, _______,
     _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
 ),    
 };
 
 uint32_t layer_state_set_user(uint32_t state) {
-  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+  return update_tri_layer_state(state, _FN, _UNICODE, _ADJUST);
 }
 
 void matrix_init_user(void) {
+    set_unicode_input_mode(UC_LNX);
```

Comment:
I recommend replacing this call (and in fact the entire `matrix_init_user()` function) with this line in your `config.h` file:
```c
#define UNICODE_SELECTED_MODES UC_LNX
```
It does the same thing as the above function, while being shorter and declarative (it also has some extra benefits; check the docs if you're interested).

Reply:
Thank You for your recomendation. I make the necesary changes.